operation/fhir-operation-everything/src/main/java/com/ibm/fhir/operation/everything/EverythingOperation.java

```diff
@@ -416,4 +485,272 @@ private static Uri uri(FHIROperationContext operationContext, String uriPath) {
                 .value(requestBaseURI + "/" + uriPath)
                 .build();
     }
+    
+    private void addIncludesSearchParameters(String compartmentType, List<String> typeParameters, MultivaluedMap<String, String> searchParameters) {
+        // Add in Location, Medication, Organization, and Practitioner resources which are pointed to
+        // from search parameters only if the request does not have a _type parameter or it does have a 
+        // _type parameter that includes these
+
+        // Remove any _includes we added last time around
+        searchParameters.remove("_include");
+        
+        // Add in _includes for all search parameters that are Location, Medication, Organization, or Practitioner
+        if (compartmentType.equals(AdverseEvent.class.getSimpleName())) {
```

Comment:
There are a lot of includes here. I'm not sure they all make sense.

Reply:
Lee, Paul, and I talked and agreed to leave this this way for now.